content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -11,29 +11,30 @@ Follow this guide to choose and set up a barcode scanner compatible with Odoo's
    :align: center
    :alt: An image of an example barcode scanner.
 
-   An image of an example barcode scanner.
-
 Scanner types
 =============
 
-Before setting up a barcode scanner, it is important to determine which scanner type best meets the
-needs of the business. There are three main types, each with their own benefits and use cases:
+Before setting up a barcode scanner, it is important to determine which type of scanner best meets
+the needs of the business. There are three main types of scanners that can be used with Odoo, each
+with their own benefits and use cases:
+
+- *USB scanners* are connected to a computer, and are suitable for businesses that scan products
+  at a fixed location, like at the checkout line in a grocery store.
 
-- **USB scanners** are connected to a computer, and are suitable for businesses that scan products
-  at a fixed location, like at the checkout in a grocery store. Ensure the chosen USB scanner is
-  compatible with the keyboard layout of the computer.
+.. important::
+   Ensure the chosen USB scanner is compatible with the keyboard layout of the computer.
```

Comment:
Can indent this admonition to be under the **USB scanners** bullet point, this also improves the list formatting by adding padding between the list items :)
```suggestion
  .. important::
     Ensure the chosen USB scanner is compatible with the keyboard layout of the computer.
```